runtime/doc/term.txt

```diff
@@ -592,6 +592,17 @@ before using the mouse:
 	"g<LeftMouse>"	is "<C-LeftMouse>	(jump to tag under mouse click)
 	"g<RightMouse>" is "<C-RightMouse>	("CTRL-T")
 
+							*bracketed-paste-mode*
+Bracketed paste mode allows users to paste lines into the terminal emulator
+without Neovim giving any special meaning to it. Most notably it won't try
```

Comment:
Maybe "underlying terminal emulator"? `TUI` seems wrong somehow, since it's not about a user interface, but the actual thing running Neovim.